etcdserver/server.go

```diff
@@ -379,6 +379,7 @@ func NewServer(cfg ServerConfig) (srv *EtcdServer, err error) {
 		cl.SetID(types.ID(0), existingCluster.ID())
 		cl.SetStore(st)
 		cl.SetBackend(be)
+		cl.SetAcceptNextStorageVersion(cfg.AcceptNextStorageVersion)
```

Comment:
`cfg.AcceptNextStorageVersion` should never change during runtime. Setting it post construction has a risk that we miss it. Could we pass it in `RaftCluster` constructor?